### ISSUE TITLE
Add default accessible label to templates using localgov_base:prev-next as needed

### ISSUE THREAD

**Issue by ctorgalson** [localgovdrupal/localgov_base#805]
Currently `localgov_base` lacks these--IIRC, we removed one from one of the consuming templates when componentizing the Prev-Next nav, but didn't add it back. We can add them by default.

There are recent fixes to the component's `prevNextAttributes` variable that can enable us to add e.g. `aria-label` and `aria-labelledby`. Using `aria-label` is thus possible immediately, but following Drupal's usual `<nav>` convention is not currently possible.

**Comment by tonypaulbarker** [comment]
These changes were merged. Closing.
